cmd/duffle/credential_generate.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/deislabs/duffle/pkg/bundle"
 	"github.com/deislabs/duffle/pkg/credentials"
 	"github.com/deislabs/duffle/pkg/duffle/home"
+	"sort"
```

Comment:
This makes the linter unhappy:

```
golangci-lint run --config ./golangci.yml
cmd/duffle/credential_generate.go:20: File is not `goimports`-ed (goimports)
```

Reply:
Ahh yes, setting up the linter now. I'll resubmit this PR soon!